fx2ait/fx2ait/converters/ait_converters.py

```diff
@@ -242,17 +246,20 @@ def acc_ops_linear(
     name: str,
 ) -> ConverterOutput:
     input_val = kwargs["input"]
-
+    if USE_ROCM:
+        shape = input_val._attrs["shape"]
+        input_val = input_val if len(shape) == 2 else reshape()(input_val, [-1, shape[-1]])
     weight = kwargs["weight"]
     assert isinstance(weight, AITTensor)
-
+    
     result = gemm_rcr()(input_val, weight)
 
     bias = kwargs["bias"]
     if bias is not None:
         assert isinstance(bias, AITTensor)
         result = elementwise(FuncEnum.ADD)(result, bias)
-
+    if USE_ROCM:
+        result = result if len(shape) == 2 else reshape()(result, [shape[0], -1, result._attrs["shape"][-1]])
```

Comment:
The result reshape seems not correct. It assumes shape==2 or 3. The shape could be larger than 3. We need the shape=shape[0:-1] + result._attrs["shape"][-1]

Reply:
On ROCM, the shape 3 is reshaped to shape 2 to calculate gemm then it is reshaped to shape 3  for output.

Reply:
what about the case where input shape is larger than 3?